flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/PendingCheckpoint.java

```diff
@@ -90,6 +90,8 @@
 
     private final Map<ExecutionAttemptID, ExecutionVertex> notYetAcknowledgedTasks;
 
+    private final List<ExecutionVertex> runningTasks;
```

Comment:
Consistent with my other comment, these should still be named `tasksToCommitTo` for now.

Reply:
I updated all the name to `tasksToCommitTo`~